modules/tensor_mechanics/include/postprocessors/AveragePointSeparation.h

```diff
@@ -0,0 +1,55 @@
+//* This file is part of the MOOSE framework
+//* https://www.mooseframework.org
+//*
+//* All rights reserved, see COPYRIGHT for full restrictions
+//* https://github.com/idaholab/moose/blob/master/COPYRIGHT
+//*
+//* Licensed under LGPL 2.1, please see LICENSE for details
+//* https://www.gnu.org/licenses/lgpl-2.1.html
+
+#pragma once
+
+#include "GeneralPostprocessor.h"
+
+// Forward Declarations
+class AveragePointSeparation;
+
+template <>
+InputParameters validParams<AveragePointSeparation>();
+
+/**
+ * Compute the value of a variable at a specified location.
+ *
+ * Warning: This postprocessor may result in undefined behavior if utilized with
+ * non-continuous elements and the point being located lies on an element boundary.
+ */
+class AveragePointSeparation : public GeneralPostprocessor
+{
+public:
+  static InputParameters validParams();
+
+  AveragePointSeparation(const InputParameters & parameters);
+
+  virtual void initialize() override {}
+  virtual void execute() override;
+  virtual void finalize() override {}
+  virtual Real getValue() override;
+
+protected:
+  // std::vector<MooseVariable *> _disp_var;
+
```

Comment:
```suggestion
```

remove

Reply:
This file is no longer in the commit now.